### PR TITLE
chore: update react-intl to latest

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Updates to latest (beta) version of `react-intl`.

Thanks to @emmenko and his [PR](https://github.com/commercetools/merchant-center-application-kit/pull/897/files)

### CODE REVIEW

**Comment by tdeekens** [comment]
Looks sharp. Be aware though that the update of app-kit with this, may it be another reason potentially, broke the MC when using the canary.

**Comment by montezume** [comment]
@tdeekens well since `react-intl` is just a peer dep / dev dep of ui-kit, this update mostly just affects our tests and our storybook right? We could still use the older version on the MC...

**Comment by tdeekens** [comment]
You are right.

**Comment by montezume** [comment]
Well I guess I'm wrong if we start relying on some new feature of react-intl beta post 14 in UI-Kit but MC is running 14? 🤷‍♂

File: package.json [hunk 1]
```diff
@@ -180,7 +180,7 @@
     "rcfile": "1.0.3",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "react-intl": "3.0.0-beta.15",
+    "react-intl": "3.0.0-beta.19",
```
Comment (emmenko) [inline package.json]: You need to update the peer dep as well. Check it out in appkit

**Comment by emmenko** [comment]
FYI: I updated to the latest version `3.0.0-beta-20`

**Comment by montezume** [comment]
@emmenko genau!

File: yarn.lock [hunk 2]
```diff
@@ -8043,32 +8043,28 @@ interpret@1.2.0, interpret@^1.0.0, interpret@^1.2.0:
...
   version "2.1.3"
   resolved "https://registry.yarnpkg.com/intl-messageformat-parser/-/intl-messageformat-parser-2.1.3.tgz#c918e36a08bd7ff01349d8ea3736262c986aaeb1"
   integrity sha512-YOP2GuQ8Y+f3j2Vti1/abcozYlkv/WsIffsRP8cgtN4Oha5hjhNnucxzyfmeAoDgDHw4s/lNyLylB01369a5lg==
 
-intl-messageformat@^4.3.0:
-  version "4.4.0"
-  resolved "https://registry.yarnpkg.com/intl-messageformat/-/intl-messageformat-4.4.0.tgz#aa196a4d04b573f4090bc417f982d81de4f74fad"
-  integrity sha512-z+Bj2rS3LZSYU4+sNitdHrwnBhr0wO80ZJSW8EzKDBowwUe3Q/UsvgCGjrwa+HPzoGCLEb9HAjfJgo4j2Sac8w==
+intl-messageformat@^5.4.2:
+  version "5.4.2"
```
Comment (emmenko) [inline yarn.lock]: Let’s regenerate the lock file so that we get the latest version here

Comment (montezume) [reply yarn.lock]: sure, done

**Pull request merged**